### PR TITLE
ANMN trigger

### PR DESCRIPTION
**Pull request by danfruehauf**

### CODE REVIEW

**Comment by danfruehauf** [comment]
@mhidas Untested, but should probably be OK.

@jonescc Mind having a look to see if there're any "lozifications" here?

**Comment by lbesnard** [comment]
the only lozified one is anmn dar yon, and I'm onto it

File: workspace/ANMN_WAVE/process/Subjobs/harvest_adcp_0.1.item [hunk 1]
```diff
@@ -1056,7 +1057,7 @@
   </node>
   <node componentName="tJavaRow" componentVersion="0.101" offsetLabelX="0" offsetLabelY="0" posX="320" posY="-160">
     <elementParameter field="TEXT" name="UNIQUE_NAME" value="tJavaRow_1"/>
-    <elementParameter field="MEMO_JAVA" name="CODE" value="// code sample:&#xA;//&#xA;// multiply by 2 the row identifier&#xA;// output_row.id = input_row.id * 2;&#xA;//&#xA;// lowercase the name&#xA;// output_row.name = input_row.name.toLowerCase();&#xA;&#xA;context.url=row5.url;"/>
+    <elementParameter field="MEMO_JAVA" name="CODE" value="// code sample:&#10;//&#10;// multiply by 2 the row identifier&#10;// output_row.id = input_row.id * 2;&#10;//&#10;// lowercase the name&#10;// output_row.name = input_row.name.toLowerCase();&#10;&#10;context.base + &quot;/&quot; + context.url=row5.url;"/>
```
Comment (jonescc) [inline workspace/ANMN_WAVE/process/Subjobs/harvest_adcp_0.1.item]: Heres one

File: workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item [hunk 2]
```diff
@@ -114,7 +115,7 @@
   </node>
   <node componentName="tJavaRow" componentVersion="0.101" offsetLabelX="0" offsetLabelY="0" posX="320" posY="-160">
     <elementParameter field="TEXT" name="UNIQUE_NAME" value="tJavaRow_1"/>
-    <elementParameter field="MEMO_JAVA" name="CODE" value="// code sample:&#xA;//&#xA;// multiply by 2 the row identifier&#xA;// output_row.id = input_row.id * 2;&#xA;//&#xA;// lowercase the name&#xA;// output_row.name = input_row.name.toLowerCase();&#xA;&#xA;context.url=row5.url;"/>
+    <elementParameter field="MEMO_JAVA" name="CODE" value="// code sample:&#10;//&#10;// multiply by 2 the row identifier&#10;// output_row.id = input_row.id * 2;&#10;//&#10;// lowercase the name&#10;// output_row.name = input_row.name.toLowerCase();&#10;&#10;context.base + &quot;/&quot; + context.url=row5.url;"/>
```
Comment (jonescc) [inline workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item]: And another

File: workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item [hunk 3]
```diff
@@ -139,7 +140,7 @@
     <elementParameter field="QUERYSTORE_TYPE" name="QUERYSTORE" value="&quot;&quot;"/>
     <elementParameter field="TECHNICAL" name="QUERYSTORE:REPOSITORY_QUERYSTORE_TYPE" value=""/>
     <elementParameter field="TECHNICAL" name="QUERYSTORE:QUERYSTORE_TYPE" value="BUILT_IN"/>
-    <elementParameter field="MEMO_SQL" name="QUERY" value="&quot;delete from timeseries where file_id = &quot; + context.fileId + &quot;; &#xA;delete from visualisation_awac where file_id = &quot; + context.fileId + &quot;; &#xA; delete from measurement_awac where file_id = &quot; + context.fileId + &quot;;&quot;"/>
+    <elementParameter field="MEMO_SQL" name="QUERY" value="&quot;delete from timeseries where file_id = &quot; + context.fileId + &quot;; &#10;delete from visualisation_awac where file_id = &quot; + context.fileId + &quot;; &#10; delete from measurement_awac where file_id = &quot; + context.fileId + &quot;;&quot;"/>
```
Comment (jonescc) [inline workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item]: And another

Comment (jonescc) [inline workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item]: Actually - this ones OK - just the two above - assume Loz wrote this one?

Comment (lbesnard) [inline workspace/ANMN_WAVE/process/Subjobs/harvest_awac_0.1.item]: I guess so :)

**Comment by mhidas** [comment]
@danfruehauf I will test with the pipelines in the PO box.

**Comment by danfruehauf** [comment]
You'll have to export each individual harvester you want to test + deploy it. If you need help, please let me know.

**Comment by danfruehauf** [comment]
@mhidas Pushed the 2 other harvesters.

@jonescc Can you please verify there are no lozifications on 90df4b4 and 4d968bf - I rebased and used your new modifications to the migration script.

**Comment by jonescc** [comment]
Looks good to me.

**Comment by danfruehauf** [comment]
> Looks good to me.

Cheers :beers:

**Comment by mhidas** [comment]
@danfruehauf Thanks! I will test.

Also, I've added a change to make the ANMN CTD profiles harvester follow the context naming conventions.

**Comment by danfruehauf** [comment]
> Also, I've added a change to make the ANMN CTD profiles harvester follow the context naming conventions.

Wicked. So we can amend the talend config. Come and talk to me when it's relevant.

**Comment by mhidas** [comment]
@danfruehauf Please merge #362 before merging this.

**Comment by mhidas** [comment]
Apart from that, they all seem to work ok!

**Comment by danfruehauf** [comment]
> Apart from that, they all seem to work ok!

:+1: We're getting closer!

**Pull request merged**